src/app/Components/email-preview/email-preview.component.html

```diff
@@ -1 +1,22 @@
-<p>email-preview works!</p>
+<div class="email-preview">
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th scope="col">Sender</th>
+        <th scope="col">Subject</th>
+        <th scope="col">
+          Date
+          <i class="fas fa-sort-up" *ngIf="toggle" (click)="sortDate()"></i>
```

Comment:
when the app has started and when re-navigate to a box, the Date column shows the down-caret (which means the Date is being sorted descending), but it's not.
solutions: don't show any carets at first or sort asc/desc in the beginning.
![image](https://user-images.githubusercontent.com/47804324/164281706-d0cb6ff3-62da-4c35-9297-3adb19763bd9.png)